integration-tests/account/remove_zero_balance.move

```diff
@@ -0,0 +1,24 @@
+//# init -n dev
+
+//# faucet --addr alice --amount 0
+
+//# run --signers alice
+script {
+    use StarcoinFramework::STC::{STC};
+    use StarcoinFramework::Signer;
+    use StarcoinFramework::Account;
+    use StarcoinFramework::Token;
+
+    fun main(account: signer) {
+        let addr: address = Signer::address_of(&account);
+
+        let coin = Token::zero<STC>();
+        Account::deposit_to_self<STC>(&account, coin);
+
+        Account::remove_zero_balance<STC>(&account);
+
+        Account::set_auto_accept_token(&account, false);
+        assert!(!Account::is_accept_token<STC>(addr), 101);
+    }
+}
+// check: EXECUTED
```

Comment:
the check statement is no longer needed. We check the transaction status with the exp file

Reply:
I am not sure what you mean. You mean the `assert` statement is no longer needed?

Reply:
"// check: EXECUTED"  is no longer needed.

Reply:
Maybe keeping the "// check: EXECUTED" statement can help read the code. It is ok.